src/pipe_mgr/shared/dal/dpdk/pipe_mgr_dpdk_ctx_util.c

```diff
@@ -103,7 +104,13 @@ int pipe_mgr_dpdk_encode_match_key_and_mask(
 			/* Copy to entry. */
 			memcpy(&entry->key_mask[offset], (uint8_t *)&val,
 					match_bytes);
+		}else if (entry->key_mask &&
```

Comment:
1. Give space between } and else. "} else"
2. Is it possible to optimize the and condition by removing the 2nd condition **_match_fields->match_type == PIPE_MGR_MATCH_TYPE_EXACT_**. If it is coming to  else part then it is implicit it is **_PIPE_MGR_MATCH_TYPE_EXACT_**

Reply:
taken care